recipes/asl/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "asl" %}
```

Comment:
I was mentioned above as the debian IPOPT maintainer, and that is incorrect. I did update it in debian once or twice but I'm not actively maintaining it.